deployment/arm/region.json

```diff
@@ -423,7 +423,7 @@
             "properties": {
                 "loggerType": "applicationInsights",
                 "credentials": {
-                    "instrumentationKey": "{{Logger-Credentials-5df9447791f0e91e700024ea}}"
+                    "instrumentationKey": "[reference(resourceId('Microsoft.Insights/components', parameters('appInsightsName')), '2015-05-01').InstrumentationKey]"
```

Comment:
This is the same line from @mydiemho 's PR.

Reply:
Should we setup app insight at all? I know we want users to choose and setup their own monitoring solution , it could be app insight, New relic, etc.

Reply:
For the time being let's keep it in and then we can remove the pieces that we want the users to setup.